gbpservice/nfp/lib/transport.py

```diff
@@ -168,7 +170,8 @@ def __init__(self, topic):
 
 def send_request_to_configurator(conf, context, body,
                                  method_type, device_config=False,
-                                 network_function_event=False):
+                                 network_function_event=False,
+                                 override_backend=None):
```

Comment:
If TCP BACKEND is set for visibility APIs even proxy_agent should use TCP BACKEND not UNIX REST BACKEND